locales/en-US/app.ftl

```diff
@@ -41,6 +41,20 @@ AppViewRouter--error-message-public =
 AppViewRouter--error-message-from-url =
     .message = Could not download the profile.
 
+# This error message is displayed when a Safari-specific error state is encountered.
+# Importing profiles from URLs such as http://127.0.0.1:someport/ is not possible in Safari.
+# https://profiler.firefox.com/from-url/http%3A%2F%2F127.0.0.1%3A3000%2Fprofile.json/
+AppViewRouter--error-message-from-localhost-url-safari =
+    .title = Safari cannot import local profiles
+    .message =
+        Due to a Safari-specific limitation, { -profiler-brand-name } is unable
+        to import profiles from the local machine in this browser.
```

Comment:
Do we need all this information in the message? 

Could we simplify to this, and link "Safari-specific limitation" to the actual webkit bug? Also, `Firefox` -> `-firefox-brand-name`

```
        Due to a <a>specific limitation in Safari</a>, { -profiler-brand-name } is unable
        to import profiles from the local machine in this browser. Please open 
        this page in { -firefox-brand-name } or Chrome instead.
```

Reply:
Thanks, that sounds good! There is a small problem with inserting the link: `<Localized>` only supports elements for values, not for attributes, and this text is currently in a .message attribute. But I can change things around so that it's a value.

Reply:
Maybe, instead of all this complex thing with `toParagraph` (probably my fault 🙈 but that was before localization), we could pass some JSX, and then it would be a lot easier to localize. What do you think?

Actually we could even remove `message`/`additionalMessage`, and just pass everything as children of `ProfileRootMessage`. That would be less odd.

Reply:
I've changed the main message to be the value of the component, and removed `toParagraph`. Do I need to change the localization string IDs, now that I've moved the contents from the message attribute to the value?

I didn't follow the last suggestion of integrating the additionalMessage into the children as well. I made it a React.Node prop, but it's still a separate property.

Reply:
Yes, it requires new IDs: https://mozilla-l10n.github.io/documentation/localization/making_string_changes.html#general-guidelines

Reply:
Ok, I've removed the word "message" from all affected IDs.